### ISSUE TITLE
TA/OPR 9750 MussaKone MiniProject #04

### ISSUE THREAD

**Issue by mussakone** [mussakone/STA9750-2024-FALL#5]
Hi @michaelweylandt!

https://mussakone.github.io/STA9750-2024-FALL/mp04.html

http://localhost:4409/mp04.html

**Comment by michaelweylandt** [comment]
Thanks @mussakone !

Peer Feedback:

- @bleuuuz
- @Cdiallo77
- @plnrbrt
- @jasonamey

It is now time for the peer feedback round for Mini-Project 04. Please
review @mussakone's submission for this mini-project and provide peer feedback.

Using the rubric at https://michael-weylandt.com/STA9750/miniprojects/mini04.html#rubric,
please grade their submission out of a total of 50 points. Note that this
rubric is slightly different than that used for Mini-Projects 01 -- 03.

For each of the five categories, please give them a separate score and provide
a total (sum) score across the entire assignment. Feel free to assign extra
credit if you feel it is warranted (following the rubric).

If you give a score of less than 5 for any category, please provide a suggestion
for improvement. (You can also give suggestions for any element they did well -
more feedback is always great!)

As you go through this peer feedback exercise, think about what you particularly
like about this submission and how you can incorporate that approach in your future
work. If something is particularly insightful or creative, give some kudos!

Evaluators: This should take you around 15 minutes per peer feedback. You are
not required to engage in substantial back-and-forth with @mussakone, but you are of course
welcome to initiate a discussion.

@mussakone: please engage fully with your peers. They are here to help you!

Submission URL should be: https://mussakone.github.io/STA9750-2024-FALL/mp04.html

Feel free to link to other repos, the course documentation, or other useful
examples.

Thanks!
@michaelweylandt

CC: @charles-ramirez

**Comment by bleuuuz** [comment]
Hi Mussakone,

Thanks for sharing and congrats on completing!

Written Communication: 7.5/10
Although the text guides the reader through your thought process, certain
parts of your report are hard to read. The first paragraph has a few
grammatical errors like "faculty that has been hired by CUNY has a an
important". Although I understand what you are trying to say the
grammatical errors make it hard to read. Along with that in task 3 under
your table collected is misspelled. Aside from that, great job on the
written portion!

Project Skeleton: 10/10
All tasks complete, nice job!

Formatting and Display: 8/10
Nice job with the graphics! You could improve some of the tables by
renaming the columns and rounding some values to make it easier for your
audience. Also maybe there is a better way to display your last two
graphics as the way they are shown make it hard to read and interpret.

Code Quality: 9/10
Your code is commented well and looks super clean! Maybe you can add in a
warnings clause so they don't show on the readers end of the website. I.E
when you load tidyverse.

Data Preparation: 10/10
Nice job! API works and data is pulled.

Overall: 44.5/50

**Comment by Cdiallo77** [comment]
Hi @mussakone, Please find my feedback below:

Written Communication: 9.5
Well done analysis, I wish you explained the bootstrap method findings. 

Project Skeleton: 10
Your report clearly answers all tasks in the mini project guidelines. 

Formatting & Display: 9.5
Your visualizations are clear and easy to understand, however you should get rid of the warnings . 

Code Quality: 10
Your code is easily understandable. 

Data Preparation: 10


Total: 49/50

**Comment by plnrbrt** [comment]
Hi @mussakone, 
Thank you so much for sharing your mini-project with us! Here are my feedbacks:

Written Communication: 9/10 I like how you put a phrase to explain the steps. Would have been nice to have maybe a phrase at the begining of each table, I see some tables that are one after the other and it is hard to follow the flow.

Project Skeleton: 10/10 All the professor tasks have been completed.

Formatting & Display: 9/10 Some of your warnings are showing as well results of package downloads. You can get rid of that by coding at the beginning of your code chunk ```{r, echo = TRUE, message = FALSE, warning = FALSE}

Code Quality: 10/10 Your code looks good and you have described it correctly to give context to the analysis code.

Data Preparation: 10/10 You used both data from FRED and AlphaVantage correctly and download it successfully.

Total: 48/50

**Comment by jasonamey** [comment]
Hello @mussakone <https://github.com/mussakone> , here is my review for
your MP-04

*Written Communication *9/10 You're writing is clear.  There are few
instances I would expand a little bit on your thinking.

*Project Skeleton *10/10 You handle all the tasks and complete the
assignments.

*Formatting and Display* 9/10 Many of your plots look good. I really like
your concluding plot of remaining ORP balances. It looks like in some
instances you suppressed the output warnings. I think you can mute much of
the unwanted output with {r message=FALSE, warning=FALSE}.

*Code Quality *10/10 I don't see issues with your code and I find it easy
to read and follow. I learned something when I looked this up :

final_data <- final_data |>
  arrange(date) |>  # Sort by date
  fill(everything(), .direction = "downup")

This is really clean and nice to read:

  us_equity = ibm_data |> rename(adjusted_close_us = adjusted_close),
  intl_equity = intl_equity |> rename(adjusted_close_intl = adjusted_close),
  wage_growth = wage_growth |> rename(wage_growth = value),
  inflation = inflation |> rename(inflation = value),
  bond_returns = bond_returns |> rename(bond_returns = value)

*Data Preparation : *This relates to some of the code selected above, but
the data collection section of your code is nice.

48 / 50
